core/celery/worker.py

```diff
@@ -0,0 +1,35 @@
+# This file provides the entry point for the Celery worker. When running the Celery worker from the
+# command line. You can use the following command:
+# celery -A "core.celery.worker.app" worker
+
+import importlib
+from pathlib import Path
+from typing import Any
+
+from celery.signals import setup_logging
+
+from core.service.container import container_instance
+
+
+def import_celery_tasks() -> None:
+    tasks_path = Path(__file__).parent / "tasks"
+    for task_file in tasks_path.glob("*.py"):
+        if task_file.stem == "__init__":
+            continue
+        module = f"core.celery.tasks.{task_file.stem}"
+        importlib.import_module(module)
```

Comment:
This does some programmatic imports to make sure that all our tasks are imported when starting the worker processes. This assumes that all our tasks will live in the `core.celery.tasks` package.